cogs/TextFilter.py

```diff
@@ -0,0 +1,701 @@
+    embed = create_default_embed(ctx)
+    embed.title = "Koala Moderation - Filtered Words"
+    if not all_words and not all_types and not all_regex:
+        embed.add_field(name="No words found", value="For more help with using the Text Filter try k!help TextFilter")
+    else:
+        embed.add_field(name="Banned Words", value=all_words)
+        embed.add_field(name="Filter Types", value=all_types)
+        embed.add_field(name="Is Regex?", value=all_regex)
+    return embed
+
+
+def create_default_embed(ctx):
+    """
+    Creates a default embed that all embeds share
+    :param ctx: The discord context
+    :return embed with basic information which should be built upon:
+    """
+    embed = discord.Embed()
+    embed.colour = KOALA_GREEN
+    embed.set_footer(text=f"Guild ID: {ctx.guild.id}")
+    return embed
+
+
+def build_channel_list_embed(self, ctx, channels):
+    """
+    Builds the embed that is sent to list all the mod channels
+    :param ctx: The discord context
+    :param channels: List of channels in the guild
+    :return embed with list of mod channels:
+    """
+    embed = create_default_embed(ctx)
+    embed.colour = KOALA_GREEN
+    embed.set_footer(text=f"Guild ID: {ctx.guild.id}")
+    embed.title = "Koala Moderation - Mod Channels"
+    embed = build_channel_list(self, channels, embed)
+    return embed
+
+
+def build_ignore_list_embed(self, ctx, channels):
+    """
+    Builds the embed to list all ignored
+    :param ctx: The discord context
+    :param channels: List of ignored users/channels
+    :return embed with list of mod channels:
+    """
+    embed = create_default_embed(ctx)
+    embed.colour = KOALA_GREEN
+    embed.set_footer(text=f"Guild ID: {ctx.guild.id}")
+    embed.title = "Koala Moderation - Ignored Users/Channels"
+    embed = build_ignore_list(self, channels, embed)
+    return embed
+
+
+def build_moderation_deleted_embed(message):
+    """
+    Builds the embed that is sent after a message is deleted for containing a banned word
+    :param message: the message object to be deleted
+    :return embed with information about the deleted message:
+    """
+    embed = create_default_embed(message)
+    embed.title = "Koala Moderation - Message Deleted"
+    embed.add_field(name="Reason",value="Contained banned word")
+    embed.add_field(name="User",value=message.author.mention)
+    embed.add_field(name="Channel",value=message.channel.mention)
+    embed.add_field(name="Message",value=message.content)
+    embed.add_field(name="Timestamp",value=message.created_at)
+    return embed
+
+
+def does_word_exist(self, ft_id):
+    """
+    Checks if word exists in database given an ID
+    :param ft_id: filtered text id of word to be removed
+    :return boolean of whether the word exists or not:
+    """
+    return len(self.database_manager.db_execute_select(
+        "SELECT * FROM TextFilter WHERE filtered_text_id = ?", args=[ft_id])) > 0
+
+
+def does_ignore_exist(self, ignore_id):
```

Comment:
If self is used, it should be in the specified class and instead use self.does_ignore_exist(ignore_id) to call this method